songbird/q2/_method.py

```diff
@@ -43,14 +43,16 @@ def multinomial(table: biom.Table,
     # split up training and testing
     trainX, testX, trainY, testY = split_training(
         dense_table, metadata, design,
-        training_column, num_random_test_examples
+        training_column, num_random_test_examples,
+        seed=0,
     )
 
     model = MultRegression(learning_rate=learning_rate, clipnorm=clipnorm,
                            beta_mean=differential_prior,
                            batch_size=batch_size,
                            save_path=None)
     with tf.Graph().as_default(), tf.Session() as session:
+        tf.set_random_seed(0)
```

Comment:
```suggestion
        tf.set_random_seed(seed)
```
this won't work - the seed will always be set to zero otherwise.

Reply:
Probably will want to have the `seed=None` option here as well.

Reply:
For the QIIME plugin, I thought it might be better to make the seed 0 always, so as to avoid exposing a seed argument to the QIIME user. Given this assumption, the code is fine as it stands (`tf.set_random_seed(seed)` will actually throw a `NameError`). Thought being that this could cut down on instances of comparing a model run with many different seeds to a baseline that was run only with the default seed, or something like that. It basically enforces the "right" behavior.

If you want me to expose the seed to the user via QIIME, that is fine too, and I can make the corresponding change.

Reply:
We don't want the seed to be always set to zero in the qiime2 side.  Thanks.

Reply:
Okay. I will rewrite so that default seed for QIIME is 0, but can be set via a parameter.

I'm not sure I understand where `seed=None` factors into this interface. I think we want the default behavior of the QIIME plugin to be that the result of two commands on the same data are directly comparable by default. Then you can change seed if you need. That shouldn't require a `seed=None` anywhere for this?

Reply:
We'll still want to have an option to not specify a seed, namely
```
if seed is not None:
    tf.set_random_seed(seed)
```